src/Tests/Sfa.Tl.ResultsAndCertification.Web.UnitTests/Controllers/AdminDashboardControllerTests/ChangeIndustryPlacementGet/When_Called_With_Valid_Data.cs

```diff
@@ -0,0 +1,98 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using NSubstitute;
+using Sfa.Tl.ResultsAndCertification.Common.Helpers;
+using Sfa.Tl.ResultsAndCertification.Web.Content.AdminDashboard;
+using Sfa.Tl.ResultsAndCertification.Web.ViewModel.AdminDashboard;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Sfa.Tl.ResultsAndCertification.Web.UnitTests.Controllers.AdminDashboardControllerTests.ChangeIndustryPlacementGet
+{
+    public class When_Called_With_Valid_Data : TestSetup
+    {
+        public int PathwayId { get; set; }
+        public IActionResult Result { get; private set; }
+        protected AdminChangeIndustryPlacementViewModel AdminChangeIndustryPlacementViewModel = null;
+        protected AdminChangeIndustryPlacementViewModel Mockresult = null;
+
+        public override void Given()
+      {
+            PathwayId = 10;
+            AdminChangeIndustryPlacementViewModel = new() { IndustryPlacementStatus = Common.Enum.IndustryPlacementStatus.Completed };
+            Mockresult = new AdminChangeIndustryPlacementViewModel
+            {
+                Uln = 1235469874,
+                FirstName = "firstname",
+                LastName = "lastname",
+                ProviderName = "provider-name",
+                ProviderUkprn = 58794528,
+                TlevelName = "t-level-name",
+                AcademicYear = 2020,
+                DisplayAcademicYear = "2021 to 2022",
+                IndustryPlacementStatus = Common.Enum.IndustryPlacementStatus.Completed
+            };
+
+            AdminDashboardLoader.GetAdminLearnerRecordAsync<AdminChangeIndustryPlacementViewModel>(PathwayId).Returns(Mockresult);
+        }
+
+        public async override Task When()
```

Comment:
Can be removed. Declared in parent class

Reply:
Done